### ISSUE TITLE
slow MEI validation/page loads

### ISSUE THREAD

**Issue by annamorphism** [DDMAL/Neon#652]
When highlighting is turned on, initial page scrolls/MEI validation take only about a second or two. As one continues using the manuscript, verification times get longer and longer, and eventually take as much as 50 seconds (even on pages that previously loaded much faster.)

**Comment by JoyfulGen** [comment]
It sounds like this issue has to do with viewing an entire manuscript in Neon, which I don't think has been done since I've been at the lab... I'll close this issue and open another one about manuscript viewing in general if necessary.

**Comment by annamorphism** [comment]
@JoyfulGen yes please do. I did a certain amount of testing on the sample manuscript in 2021, but we switched to individual pages to be part of a better workflow that summer, and the manuscript viewer function has been neglected ever since (see e.g. https://github.com/DDMAL/Neon/issues/1002)...
